src/library/trackmodel.h

```diff
@@ -62,7 +62,7 @@ class TrackModel {
         SORTCOLUMN_YEAR = 5,
         SORTCOLUMN_GENRE = 6,
         SORTCOLUMN_COMPOSER = 7,
-m        SORTCOLUMN_GROUPING = 8,
```

Comment:
This looks like a potential compilation error. All commits should be compilable without errors. If you accidently commited a broken state, please fixup the erroneous commits and force-push.

Reply:
Already fixed!

Reply:
Yes, but if I'm doing a git bisect and try to build commit https://github.com/mixxxdj/mixxx/pull/3167/commits/5fdcb305ae3de40083c343b8835abbb3029f9a03 it will fail. Please run:

git rebase -i HEAD~2 and replace `pick` in front of `a20bddd` with `fixup`.

Reply:
Done!